source/fundamentals/data-formats/document-data-format-extended-json.txt

```diff
@@ -291,14 +291,13 @@ expressions, to simplify the "Relaxed" JSON output.
 
    System.out.println(doc.toJson(settings)));
 
-The output of this code should resemble the following:
+The output of this code should look something like this:
 
 .. code-block:: json
 
    {"_id": "507f1f77bcf86cd799439012", "createdAt": "2020-09-30T21:00:09Z", "myNumber": 4794261}
 
-Without specifying the converters, the "Relaxed" JSON output resembles the
-following:
+Without specifying the converters, the "Relaxed" JSON output should look something like this:
```

Comment:
Should this be `Relaxed mode JSON output` instead similar to the other updated versions?

Reply:
Thanks for the note Chris. I think I left that there as the server docs seem to use both, but I think it makes sense for all instances in the document to be the same. Updated here: https://github.com/mongodb/docs-java/pull/57/commits/cb5e031d44b5bb8c4baab131b5e2cf3a03ae33f4